### ISSUE TITLE
Wrong fonts on profiling pane and maximized help pane on macOS Catalina

### ISSUE THREAD

**Issue by SantiRaposo** [rstudio/rstudio#5543]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.5001
    OS Version      :  MacOS Catalina (10.15)
    R Version       :  3.6.1

### Steps to reproduce the problem
Run the following piece of code on Rstudio:
```
library(profvis)
profvis({
  data(diamonds, package = "ggplot2")
  
  plot(price ~ carat, data = diamonds)
  m <- lm(price ~ carat, data = diamonds)
  abline(m, col = "red")
})
```

### Describe the problem in detail
The profiling window has a serifed font, not the one used by any other windows. This also happens when you maximize the help pane.
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/12662446/66664197-5e94f200-ec4c-11e9-99b0-c70757118e17.png">
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/12662446/66664222-6eacd180-ec4c-11e9-8c75-a5706d2143e6.png">

### Describe the behavior you expected
I expected the font to be the same as the one used by RStudio in other panes.

**Comment by kevinushey** [comment]
Thanks for the bug report! This should be fixed in the latest [daily builds](https://dailies.rstudio.com) of RStudio, and we hope to backport a fix to the RStudio v1.2 release as soon as possible.

**Comment by jmcphers** [comment]
Now fixed in the dailies and backported to 1.2.
